### ISSUE TITLE
Add CI

### ISSUE THREAD

**Issue by raxod502** [raxod502/radian#319]
By this point, there are a number of ways I can accidentally drop the ball on my Emacs config and push something stupid:

* forget to test that a deferred installation actually works
* forget to freeze my package versions
* forget to validate my el-patches
* fail to notice that Emacs errors out on startup (yes, really)

This stuff should be tested by Travis so I can't get away with it.

**Comment by raxod502** [comment]
This won't be doable, of course, until [deferred installation actually works in the first place](https://github.com/raxod502/radian/issues/315) and [we have a functioning package manager](https://github.com/raxod502/radian/issues/324).

Probably we don't have to wait on Dotman, though; a targeted installation might be more appropriate, since trying to support ancient Ubuntu versions is most likely an exercise in futility. Rolling release, people. Rolling release.

**Comment by raxod502** [comment]
For `straight.el` I'm using https://github.com/rejeep/evm to install Emacs on Travis. It works pretty well.

**Comment by raxod502** [comment]
Honestly, this is way overkill for a personal configuration. It's not like I don't notice when things break, and there's always `master` which I attempt to keep stable.
